test/common/alerts_firing.go

```diff
@@ -115,8 +115,15 @@ func TestIntegreatlyAlertsFiring(t TestingTB, ctx *TestingContext) {
 	//fail immediately if one or more alerts have fired
 	if err := getFiringAlerts(t, ctx); err != nil {
 		podLogs(t, ctx)
-		t.Skipf("Skipping due to known issue : %v, reported in Jira: https://issues.redhat.com/browse/MGDAPI-1404", err.Error())
-		//t.Fatal(err.Error())
+		isInProw, e := isInProw(ctx)
+		if e != nil {
+			t.Logf("error getting in_prow annotation: %s", e)
+		}
+		if isInProw {
+			t.Skipf("Skipping due to known issue in Prow, UIBBT alests fire there: %v, reported in Jira: https://issues.redhat.com/browse/MGDAPI-5193", err.Error())
+		} else {
+			t.Fatal(err.Error())
```

Comment:
do we want this test to run at as part of functionals? If so, it looks to me like we are going to error out on non-prow runs

Reply:
Yes, we do. I can validate this via pipeline to be 100% sure but `err` is only populated via `getFiringAlerts` (line 116) and if it is non-null then we should error because under normal circumstances (non-prow) there should be no alerts firing. If there is an error when calling `isInProw` then `e` is populated (not `err`) so that should not affect anything

Reply:
Hey @MStokluska I executed both tests (C01 and "Verify Alerts are not firing during or after installation apart from DeadMansSwitch") against the cluster (using test/e2e/single make target) and both passed fine. I tried with `in_prow` set to false and also with `in_prow` missing completely.